client/src/hooks/use-router.tsx

```diff
@@ -76,6 +83,13 @@ export function useRouter() {
                 return { Component: Configuration };
               },
             },
+            {
+              path: 'query',
```

Comment:
Why added twice?

Reply:
Removed the extr add was probably a merge mistake